### DISCUSSION TITLE
Only Repo name displaying on my website

### DISCUSSION THREAD

**Discussion by Waltrex01** [community/community#70283] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Feature Area

Issues

### Body

Please help!, i've been working on a website for a while now
uploaded it on github but only the repo name is showing, what should i do?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Waltrex01 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
